packages/dd-trace/src/service-naming/schemas/v0/storage.js

```diff
@@ -57,6 +57,7 @@ const storage = {
         pluginConfig.service || `${tracerService}-elasticsearch`
     },
     ioredis: redisConfig,
+    iovalkey: redisConfig,
```

Comment:
This has to also be replaced. The command is not correct.
We could just make it a method and accept a name as parameter that defaults to redis.